scripts/ci/setenv.sh

```diff
@@ -1,9 +1,46 @@
 #!/usr/bin/env bash
 set -euxo pipefail
 
-# Install Go using the same travis approach
-echo "Installing ${GO_VERSION} with gimme."
-eval "$(curl -sL https://raw.githubusercontent.com/travis-ci/gimme/master/gimme | GIMME_GO_VERSION=${GO_VERSION} bash)"
+# Install Golang using gvm
+echo "Installing ${GO_VERSION} with gvm."
+MSG="environment variable missing"
+GO_VERSION=${GO_VERSION:?$MSG}
+HOME=${HOME:?$MSG}
+OS=$(uname -s| tr '[:upper:]' '[:lower:]')
+ARCH=$(uname -m| tr '[:upper:]' '[:lower:]')
+GVM_CMD="${HOME}/bin/gvm"
+
+if command -v go ; then
```

Comment:
Question: Does [`g`](https://github.com/stefanmaric/g) can be a good alternative?

Reply:
Since `gvm` is maintained by @andrewkroh and it's well used for a few number of years by the ingest team, I'd say we can keep using `gvm` for now.

Probably, we could contribute to the `gvm` and facilitate the installation with a shell/bash script so all this magic won't be needed

Reply:
https://github.com/andrewkroh/gvm/pull/67 is a tiny proposal